code/__HELPERS/time.dm

```diff
@@ -20,6 +20,11 @@
 /proc/SQLtime(timevar)
 	return time2text(timevar || world.timeofday, "YYYY-MM-DD hh:mm:ss")
 
+/proc/station_time(display_only = FALSE, wtime=world.time)
```

Comment:
Why is this readded? I'm 90% sure we made this cached or something. Take a look at Stat() for more info

Reply:
Was added as part of pausing time since death for people in stasis.